Felienne/hedy
# [FIX] Hidden programmers mode toggle on quiz and view program

**Description**
The `programmers mode` toggle is implemented in the `incl-editor-and-output.html` file. As this file contains the code editor as well as the output, this is includes in different pages where the programmers mode is not relevant. For example when being on the quiz tab (the editor is hidden so the toggle makes no sense) and the view program page (we can't edit there so the toggle makes no sense). In this PR we fix the bug where the toggle is always visible by hidden it when necessary.

**Fix for**
This PR fixes #1434.

**How to test**
Use the adventure tabs to navigate to the quiz and back. Notice that the toggle button hides when on the quiz and returns when switching back. Also open a stored and shared program and notice that the toggle is hidden in this view as well.
